2018-10-31T10:12:00.621Z - [32minfo[39m: Data Base connection success ...
2018-10-31T10:12:00.649Z - [31merror[39m: uncaughtException: expected 0 to deeply equal 1 date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=156659712, heapTotal=122327040, heapUsed=97125696, external=1176011, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=43, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=43, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected 0 to deeply equal 1,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:43:43),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.670Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 401 but got 200 date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=157200384, heapTotal=122327040, heapUsed=100216240, external=1177021, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=75, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 401 but got 200,     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:75:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.687Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 403 but got 200 date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161255424, heapTotal=126521344, heapUsed=93171848, external=75358, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=87, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 403 but got 200,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:87:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.695Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161255424, heapTotal=126521344, heapUsed=93837544, external=84649, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=119, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"',     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:119:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.701Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161255424, heapTotal=126521344, heapUsed=94378888, external=85697, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=145, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"',     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:145:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.712Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 204 but got 200 date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161525760, heapTotal=126521344, heapUsed=96906568, external=94859, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101820, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.end, line=173, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 204 but got 200,     at chai.request.put.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:173:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.718Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 404 but got 200 date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161525760, heapTotal=126521344, heapUsed=99324040, external=95365, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101821, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.send.end, line=188, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 404 but got 200,     at chai.request.put.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:188:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.731Z - [31merror[39m: uncaughtException: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161525760, heapTotal=126521344, heapUsed=99891736, external=104565, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101821, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=224, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:224:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.734Z - [31merror[39m: uncaughtException: expected { status: false } to be empty date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161525760, heapTotal=126521344, heapUsed=100261856, external=105047, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101821, trace=[column=66, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=234, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false } to be empty,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:234:66),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:12:00.739Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:12:00 GMT+0400 (+04), pid=23323, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=161525760, heapTotal=126521344, heapUsed=100617096, external=113876, loadavg=[1.66748046875, 1.0859375, 0.953125], uptime=101821, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=250, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:250:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
